pep-0572.rst

```diff
@@ -16,8 +16,9 @@ Abstract
 ========
 
 This is a proposal for creating a way to assign to variables within an
-expression using the notation ``NAME := expr``.
-
+expression using the notation ``NAME := expr``. A new exception,
+``TargetScopeError`` is added, and there is one change to evaluation
+order.
```

Comment:
I would prefer to be more specific for "evaluation order", like "change to evaluation order of dict comprehension". Otherwise, people might be scared of a bigger change, whereas it's really a minor change ;-)

Reply:
Well let them be scared. The abstract is not the place to repeat the spec. The evaluation order change has its own section in the ToC so it's easy to find.